### PR TITLE
#463 cache alle TeamOperations

### LINKED ISSUE #463: Cache requests to team service
### Expected behavior
The teams service implemented some rate limiting recently. This leads to throttling exceptions, when people are resolving violations in a bulk.

### Step needed
Team membership doesn't change a lot -> It can be cached for some minutes.

### PR DESCRIPTION
**Pull request by harti2006**
fixes #463

### CODE REVIEW

**Comment by harti2006** [comment]
:+1:

File: fullstop/src/main/java/org/zalando/stups/fullstop/config/CacheConfig.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        return buildCacheManager("maximumSize=1000,expireAfterWrite=2h");
+    }
+
+    @Bean
+    public CacheManager oneMinuteTTLCacheManager() {
+        return buildCacheManager("maximumSize=1000,expireAfterWrite=1m");
+    }
+
+    private static CacheManager buildCacheManager(final String spec) {
+        final GuavaCacheManager cacheManager = new GuavaCacheManager();
```
Comment (gagoman) [inline fullstop/src/main/java/org/zalando/stups/fullstop/config/CacheConfig.java]: `Guava` cache would be replaced by `Caffeine` (http://docs.spring.io/spring-boot/docs/current-SNAPSHOT/reference/html/boot-features-caching.html#boot-features-caching-provider-caffeine)

Maybe this is a right time to integrate it?

Comment (harti2006) [inline fullstop/src/main/java/org/zalando/stups/fullstop/config/CacheConfig.java]: Nice hint, but it seems that it is only available in the SNAPSHOT version of Spring Boot so far.

**Comment by gagoman** [comment]
👍

**Pull request merged**